### PR TITLE
Apple TV support 5: adjustments after manual import of #10427

### PR DESCRIPTION
**Pull request by dlowder-salesforce**
**Motivation**

After reviewing changes between my PR https://github.com/facebook/react-native/pull/10427 and what was eventually manually imported to master, found two minor adjustments needed.

**Test plan**

Existing tests should still pass.

### CODE REVIEW

**Comment by dlowder-salesforce** [comment]
@javache these are the only changes I found that should be needed after merging your manual import of Apple TV code to my repo.

File: React/Base/RCTPlatform.m [hunk 1]
```diff
@@ -39,7 +39,11 @@ @implementation RCTPlatform
     @"forceTouchAvailable": @(RCTForceTouchAvailable()),
     @"osVersion": [device systemVersion],
     @"systemName": [device systemName],
+#if TARGET_OS_TV
+    @"interfaceIdiom": interfaceIdiom(UIUserInterfaceIdiomTV),
+#else
     @"interfaceIdiom": interfaceIdiom([device userInterfaceIdiom]),
```
Comment (javache) [inline React/Base/RCTPlatform.m]: I tested this manually and I don't understand why you need to TARGET_OS_TV, `[device userInterfaceIdiom]` should already return the right thing.

Comment (dlowder-salesforce) [inline React/Base/RCTPlatform.m]: I agree, but am having difficulty finding definite Apple documentation stating that [device userInterfaceIdiom] will return this value if and only if the code is running on Apple TV.  I'll replace this with a comment, so we can more easily find the issue if Apple changes things underneath us in the future....

File: React/Base/RCTPlatform.m [hunk 2]
```diff
@@ -39,6 +39,8 @@ @implementation RCTPlatform
     @"forceTouchAvailable": @(RCTForceTouchAvailable()),
     @"osVersion": [device systemVersion],
     @"systemName": [device systemName],
+    // userInterfaceIdiom should return UIUserInterfaceIdiomTV if and only if we are running on Apple TV.
+    // If this ever changes, we may need to adjust this code to avoid breakage when running on tvOS.
```
Comment (javache) [inline React/Base/RCTPlatform.m]: I don't really think we need this comment, it's really just repeating the documentation for `[device userInterfaceIdiom]`

**Comment by javache** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**